apps/docs/content/guides/functions/examples/sentry-monitoring.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Monitoring with Sentry'
+description: 'Monitor Edge Functions with the Sentry Deno SDK.'
+---
+
+Add the [Sentry Deno SDK](https://docs.sentry.io/platforms/javascript/guides/deno/) to your Supabase Edge Functions to easily track exceptions and get notified of errors or performance issues.
+
+### Prerequisites
+
+- [Create a Sentry account](https://sentry.io/signup/).
+- Make sure you have the latest version of the [Supabase CLI](https://supabase.com/docs/guides/cli#installation) installed.
+
+### 1. Create Supabase function
+
+Create a new function locally:
+
+```bash
+supabase functions new sentryfied
+```
+
+### 2. Add the Sentry Deno SDK
+
+Handle exceptions within your function and send them to Sentry.
+
+```tsx
+import * as Sentry from 'https://deno.land/x/sentry/index.mjs'
+
+Sentry.init({
+  dsn: SENTRY_DSN,
+  integrations: [],
+  debug: true,
+  // Performance Monitoring
+  tracesSampleRate: 1.0,
+  // Set sampling rate for profiling - this is relative to tracesSampleRate
+  profilesSampleRate: 1.0,
+})
+
+// Set region and execution_id as custom tags
+Sentry.setTag('region', Deno.env.get('SB_REGION'))
+Sentry.setTag('execution_id', Deno.env.get('SB_EXECUTION_ID'))
+
+Deno.serve(async (req) => {
+  try {
+    const { name } = await req.json()
+    const data = {
+      message: `Hello ${name}!`,
+    }
```

Comment:
```suggestion
    // This will throw, as `name` in our example call will be `undefined`
    const data = {
      message: `Hello ${name}!`,
    }
```

To provide better context